main/templates/registration/login.html

```diff
@@ -1,21 +1,21 @@
-{% extends "base/base.html" %}
+{% extends "base/datastore_base.html" %}
 {% load i18n %}
 
 
 {% block content %}
-    <div class="uu-inner-container">
+    <div class="uu-container">
         <div class="col-12">
             <h2>
                 {% trans 'login:header:login' %}
             </h2>
-            <form method="post">
+            <form method="post" class="uu-form uu-form-no-gap">
                 {% csrf_token %}
-                {{ form.as_p }}
+                {{ form }}
                 <button type="submit">
                     {% trans 'login:button:login' %}
                 </button>
             </form>
```

Comment:
This doesn't work correctly, as the auth form that's being used is not a ``TemplatedForm`` (as it's a default, Django supplied, form).

In the PPN I made my own HTML for this form, so you could just copy+paste that. Alternatively you could add overrrides for the default login view and form to setup the TemplatedForm stuff, but that's more work.

Reply:
Oh also, the button is missing the required classes